src/static/js/modules/prepare-worksheets/worksheet-config.js

```diff
@@ -223,7 +228,8 @@ this.worksheetDefaults = {
                 "alternative":"Think of all the little things that you are used to calling your landlord to deal with: a cracked window, a broken dishwasher, or a clogged toilet. As a homeowner, you will either have to fix these yourself or call and pay for a professional.",
                 "altText": "",
                 "explanation":"",
-                "deletable": false
+                "deletable": false,
+                "uneditable": true
```

Comment:
Although now that I think about it, the inability to edit/delete are both results of the row being required. I think I'll consolidate these as one attribute.